tests/test_command_line_interface.py

```diff
@@ -1,68 +1,96 @@
+                                                    "tests/input/sample_file.py", \
+                                                    "--function_without_docstrings"])
     assert func_wo_docstr_result.exit_code == 0
     assert "3" in func_wo_docstr_result.stdout
 
 
 def test_main_function_with_docstrings():
     """Test case to see if function without docstrings argument is correct."""
-    func_w_docstr_result = cli_runner.invoke(cli, ["--input-file", "tests/input/sample_file.py", "--function_with_docstrings"]) # pylint: disable=C0301
+    func_w_docstr_result = cli_runner.invoke(cli, ["--input-file", \
+                                                   "tests/input/sample_file.py", \
+                                                   "--function_with_docstrings"])
     assert func_w_docstr_result.exit_code == 0
     assert "3" in func_w_docstr_result.stdout
 
 
 def test_main_class_without_docstrings():
     """Test case to see if class without docstrings argument is correct."""
-    class_wo_docstr_result = cli_runner.invoke(cli, ["--input-file", "tests/input/sample_file.py", "--class_without_docstrings"]) # pylint: disable=C0301
+    class_wo_docstr_result = cli_runner.invoke(cli, ["--input-file", \
+                                                     "tests/input/sample_file.py", \
+                                                     "--class_without_docstrings"])
     assert class_wo_docstr_result.exit_code == 0
     assert "1" in class_wo_docstr_result.stdout
 
 def test_main_class_with_docstrings():
     """Test case to see if class with docstrings argument is correct."""
-    class_w_docstr_result = cli_runner.invoke(cli, ["--input-file", "tests/input/sample_file.py", "--class_with_docstrings"]) # pylint: disable=C0301
+    class_w_docstr_result = cli_runner.invoke(cli, ["--input-file", \
+                                                    "tests/input/sample_file.py", \
+                                                    "--class_with_docstrings"])
     assert class_w_docstr_result.exit_code == 0
     assert "1" in class_w_docstr_result.stdout
 
 def test_if_statements():
     """Test case to see if 'if statements' argument is correct."""
-    if_state = cli_runner.invoke(cli, ["--input-file", "tests/input/sample_file.py", "--if_statements"]) # pylint: disable=C0301
+    if_state = cli_runner.invoke(cli, ["--input-file", \
+                                       "tests/input/sample_file.py", "--if_statements"])
     assert if_state.exit_code == 0
     assert "1" in if_state.stdout
+
+def test_main_parameters():
+    """Test case to see if function finds the parameters."""
+    find_parameter_result = cli_runner.invoke(cli, ["--input-file", \
+                                                    "tests/input/sample_file.py", \
+                                                    "--find_parameters"])
+    assert find_parameter_result.exit_code == 0
+    assert "0" in find_parameter_result.stdout
```

Comment:
Once the ability to specify the name of a function is added, this test case should be updated.